src/components/createComponent.js

```diff
@@ -0,0 +1,96 @@
+const fs = require("fs");
+const mkdirp = require("mkdirp");
+
+const componentName = process.argv[2];
+const folderName =
+  componentName.charAt(0).toLowerCase() + componentName.slice(1);
+
+/* eslint no-console: 0 */
+const createFolder = () => {
+  mkdirp(folderName, err => {
+    if (err) {
+      console.error(err);
+    }
+  });
+};
+
+const writeThis = (fileName, contents) => {
+  fs.writeFile(`./${folderName}/${fileName}.js`, `${contents}`, err => {
```

Comment:
scripts like this will need to work in Windows so should use path.join rather than '/'